core/templates/pages/exploration-editor-page/exploration-editor-page.component.spec.ts

```diff
@@ -42,6 +42,7 @@ import { PageTitleService } from 'services/page-title.service';
 import { SiteAnalyticsService } from 'services/site-analytics.service';
 import { StateTopAnswersStatsBackendApiService } from
   'services/state-top-answers-stats-backend-api.service';
+import { EventEmitter } from '@angular/core';
```

Comment:
Move this to line 19 with the other `@angular` imports.

Reply:
Done